showkase-processor/src/main/java/com/airbnb/android/showkase/processor/logging/ShowkaseValidator.kt

```diff
@@ -75,6 +77,13 @@ internal class ShowkaseValidator {
         }
     }
 
+    // This should check if it is an annotation with multiple @Preview annotations
+    private fun checkElementIsMultiPreview(element: XElement): Boolean {
+       return element.getAllAnnotations().size >= 2 && element.findAnnotationBySimpleName(
+                PREVIEW_SIMPLE_NAME) != null && element.findAnnotationBySimpleName(
+           METADATA_SIMPLE_NAME) == null
+    }
```

Comment:
I'm very unsure about this method. For me it made sense as in this step for the multi.preview annotation, it has more than or two annotations, at least there should be a preview and the metadata annotation should not be present yet 🤔 However, Im having some doubts if it is correct. Is there a better way to do this?

Reply:
Why are we looking for the Metadata annotation? Does that denote that it's an annotation?

Reply:
Also, we don't necessarily need the size >= 2 check. While that's probably going to be true in most cases, we don't need to prescribe that since it's not a requirement for multi-preview. You can annotate another annotation with a single `@Preview` so no reason to add an extra requirement.

Reply:
That is a good point! Maybe I can check its the element is an annotation class type? 🤔

Reply:
Yeah that's probably a better direction to go in